lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -1747,166 +1761,188 @@ export class UserAgentApplication {
    * @hidden
    */
   // TODO: Break this function up - either into utils or token specific --- too long to be readable
-  protected saveTokenFromHash(tokenResponse: TokenResponse): void {
-
-    this.logger.info("State status:" + tokenResponse.stateMatch + "; Request type:" + tokenResponse.requestType);
+  protected saveTokenFromHash(hash: string, stateInfo: StateResponse): AuthResponse {
+    this.logger.info("State status:" + stateInfo.stateMatch + "; Request type:" + stateInfo.requestType);
     this.cacheStorage.setItem(Constants.msalError, "");
     this.cacheStorage.setItem(Constants.msalErrorDescription, "");
 
+    let response : AuthResponse;
+    let error: AuthError;
+    const parameters = this.stringifyHash(hash);
```

Comment:
inputHashParameters?

Reply:
parameters is just a carry over. Long names will just add unnecessary complexity to readability. parameters is fine unless we start having multiple parameter types.

Reply:
hashParams?